docs/installation/server-config/index.rst

```diff
@@ -10,25 +10,21 @@ Configuring KSQL Server
     ccloud-ksql
     avro-schema
 
-KSQL configuration parameters can be set for Admin Client, KSQL Query, KSQL Server, Kafka Streams, and Kafka Clients. The
-KSQL Server parameters are configured via the ``/etc/ksql/ksql-server.properties`` file. 
+Configuration parameters can be set for KSQL queries, the Kafka Admin Client, the KSQL Server, the Kafka Streams API, the Kafka Producer, and the Kafka Consumer. 
 
-You can set the following parameters for the KSQL Server.
+These can all be configured via the ``/etc/ksql/ksql-server.properties`` file. 
```

Comment:
This is the default path for the properties file when using Confluent CLI. Technically you can have properties files in any path. Also you should be able to have multiple properties files if you are running multiple servers in a machine. I would remove the explicit path and would just mention the `properties file`.